src/components/LogoFooter.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <footer class=" fixed-bottom row justify-content-md-center">
+      <div class="footer col-md-8 justify-content-md-center">
+        <div class="col col-md-auto text-center">
+          <img src="../assets/movielogo.png" class="logo" alt="Netflix Roulette logo">
```

Comment:
make global configuration for assests folder, in many cases those folder can be changed
or webpack(define-webpack-plugin) or window object(not preferable but easier)

Reply:
could not find how to do this, solutions I found  are not working.. could you give the link to working example?